New Page About Balances & Locks (#5859)
w3f/polkadot-wiki: docs/learn/learn-polkadot-opengov.md

track. The origins and tracks parameters are preset values that set the duration of a referendum as
well as how many referenda can be voted on simultaneously. For example, a treasury proposal can now
be submitted in different tracks depending on the amount requested. A proposal for a small tip will
need to be submitted to the Small Tipper track, while a proposal requiring substantial funds will
need to be submitted to the Medium or Big Spender track.

The [Polkadot Technical Fellowship](./learn-polkadot-technical-fellowship.md) can decide to
[whitelist](#whitelisting) a proposal that will be enacted through the Whitelist Caller origin.
Those proposals will have a shorter Lead-in, Confirmation, and Enactment period when compared to the
Root Origin track.

Each track has its own preset [Approval and Support](#approval-and-support) curves based on the
origin's privileges. When both the approval and support criteria are satisfied for a specific period
(called the confirmation period), the referendum passes and will be executed after the enactment
period.

> All referenda within each track and across tracks can be voted on simultaneously (assuming the track

Reviewer: ```suggestion
All referenda within each track and across tracks can be voted on simultaneously (assuming the track's
```

{{ polkadot: 15 DOT for a 2-week :polkadot }}{{ kusama: 1.5 KSM for a 2-week :kusama }} lock period
=> {{ polkadot: 15 x 2 = 30 Votes :polkadot }}{{ kusama: 1.5 x 2 = 3 Votes :kusama }}

Even though both Logan and Kevin vote with more
{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} than Peter, the lock period for both of them
is less than Peter’s, leading to their voting power counting as less.

:::info Staked tokens can be used in governance

> While a token is locked, you can still use it for voting and [staking](./learn-staking.md). You are

Reviewer: ```suggestion
While the tokens are locked, you can still use them for voting and [staking](./learn-staking.md). You are
```
